service/create/smallcloudconfig.go

```diff
@@ -0,0 +1,41 @@
+package create
+
+import (
+	"bytes"
+	"encoding/base64"
+	"html/template"
+	"io/ioutil"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+var (
+	smallCloudconfigPath = "templates/user-data.sh"
+)
+
+type SmallCloudconfigConfig struct {
+	MachineType string
+	Region      string
+	S3Bucket    string
+	ClusterID   string
+}
+
+func (s *Service) SmallCloudconfig(config SmallCloudconfigConfig) (string, error) {
+	b, err := ioutil.ReadFile(smallCloudconfigPath)
```

Comment:
I have the abstraction issue here again. Lets please not rely on some FS in this hardcoded way. I feel like we hit this issue in every other PR. Do you guys not agree on this practice? What can we do to not see the same design "flaws" over and over again? Is my communication not good enough?